### PR TITLE
Add Device Story show page

### PR DESCRIPTION
**Pull request by rdeskins**
Here's what I have right now. Tried to have a similar layout to the realtime website, but had to put the map below to make everything fit.
image

### CODE REVIEW

**Comment by robouden** [comment]
Good progress!  
Can we have the same scaling/time selection as in Garfana?
Also possible to export the data or links for exporting the displayed data?

Rob

**Comment by rdeskins** [comment]
> Good progress!
> Can we have the same scaling/time selection as in Garfana?
> Also possible to export the data or links for exporting the displayed data?
> 
> Rob

For the scaling/time, do you mean how Grafana will show the last 2 days? I can change the CPM graph to that scaling. 
I'm not sure what you mean by the exporting part though.

**Comment by robouden** [comment]
"For the scaling/time, do you mean how Grafana will show the last 2 days" meaning select the time for the display. Not only 2 days.

For the export see 
Screenshot from 2020-08-18 09-03-23

**Comment by rdeskins** [comment]
With the Grafana iframes, i haven't found a way to add a time range selector when embedding a graph.
I see what you mean now about the exporting. I could add a link to that, but maybe it would make sense to add a link to the whole grafana page, especially since I can't find a way to add time scaling.

**Comment by robouden** [comment]
Robin,

Yes, I came across the same issue with Iframes on RT a long time ago and decided to use Worpress with links to Grafana to make things work. I think it is a necessity to use the time scaling for displaying the data.

rob

**Comment by matschaffer** [comment]
Let's not implement time scrolling on this page. Just provide a link to the grafana view if people want to dig deeper. 

The goal is to replace https://realtime.safecast.org/sensors/200041/ which doesn't include on-page time selection either.

I would just mimic that page for now.

Here's a rough sketch of how I'd lay it out.

USA__NY__Shrub_Oak___SAFECAST

We can add fields to the `device_stories` table for stuff ttserve doesn't provide, though it comes with the assumption that some moderator (or eventually device owner) will help fill those in.

**Comment by rdeskins** [comment]
This is what it looks like right now after after those changes. Imo the table still looks a bit weird, but its easier to read the measurements/dates and notice the grafana link at least.
image

File: app/views/device_stories/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        <div class="span2">
+            <table class="table table-hover">
+                <thead>
+                    <%= time_ago_in_words(@device_story.last_seen) %> ago (<%= @device_story.last_seen %>)
+                </thead>
+                <tr>
+                    <td style="font-size: 25px"><%= @device_story.last_values %></td>
+                </tr>
+            </table>
+            <a href="http://grafana-dev.safecast.cc/d/DFSxrOLWk/safecast-device-details?orgId=1&from=now-90d&to=now&refresh=15m&var-device_urn=<%= @device_story.device_urn %>"
```
Comment (matschaffer) [inline app/views/device_stories/show.html.erb:20]: We should swap this out for a configurable value, and set the default to https://grafana.safecast.cc/d/DFSxrOLWk

**Comment by Nakushi64** [comment]
Screenshot of the current state of the layout: Screenshot 2020-11-10 at 14 40 04

**Review by matschaffer — APPROVED** [review]

**Pull request merged**